app/src/examples/LayoutAnimations/OlympicAnimation.tsx

```diff
@@ -157,17 +157,18 @@ export default function OlympicAnimation() {
         <View>
           <Animated.View
             entering={blueRingAnimation}
-            exiting={blueRingExitAnimation}
-            style={{ transform: [{ translateX: -13 }] }}>
-            <Svg width={500} height={500}>
-              <Path
-                fill="none"
-                stroke="#2A5AA6"
-                strokeWidth={5}
-                strokeMiterlimit={10}
-                d="M106.9 180.7c6.3-14.9 23.4-21.9 38.3-15.6 14.9 6.3 21.9 23.4 15.6 38.3-6.3 14.9-23.4 21.9-38.3 15.6-10.8-4.6-17.9-15.2-17.9-26.9 0-3.9.8-7.8 2.3-11.4"
-              />
-            </Svg>
+            exiting={blueRingExitAnimation}>
+            <View style={{ transform: [{ translateX: -13 }] }}>
```

Comment:
Do we really need these 13px though?

Reply:
Yes, we need it. We could also move all the other rings right instead of moving this particular ring left, and that would make a better solution, since the whole logo is off-center. However I've decided not to make a big here